_help/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Help
+layout: main
+permalink: /help/
+bg_color: bg-blue
+links:
+  - name: Creating an account
+    url: /help/creating-an-account/do-i-need-a-mobile-phone-to-use-logingov/
+    img_url: /assets/img/create-account-light.svg
+  - name: Signing in
+    url: /help/signing-in/i-cant-remember-where-my-personal-key-is-and-i-dont-have-my-phone-with-me/
+    img_url: /assets/img/sign-in-light.svg
+  - name: Change settings
+    url: /help/changing-settings/how-do-i-change-my-email-address/
+    img_url: /assets/img/change-settings-light.svg
+  - name: Privacy and security
+    url: /help/privacy-and-security/can-i-remove-a-saved-password-or-login-information-from-my-browser/
+    img_url: /assets/img/privacy-security-light.svg
+---
+
+<div class="bg-navy">
+  <div class="container cntnr-xxskinny py2">
+    <h1 class="my0 white">{{ page.title }}</h1>
+  </div>
+</div>
+<div class="bg-blue flex">
+  <div class="container cntnr-xxskinny px2 pt4 pb5 w-100pc">
+    <div class="clearfix">
+      <h2 class="h4 mb2 sans-serif white">Browse common topics</h2>
+      <ul class="pl0">
+      {% for subpage in page.links %}
+        {% assign subpage_slug = subpage.name | slugify %}
+        {% assign first_subpage = site.help | where_exp: "item", "item.url contains subpage_slug" | where: 'order', 1 | first %}
+        <li class="list-style-none mb2">
+          <a href="{{ first_subpage.url | prepend: site.baseurl }}">
+            <button class="btn btn-inverse btn-primary btn-big flex flex-center w-100pc">
+              <img src="{{ subpage.img_url | prepend: site.baseurl }}" alt="" class="w-60p mr2" />
+              <span class="w-100pc left-align">{{ subpage.name }}</span>
```

Comment:
Basscss has a `flex-auto` class that will allow an element to automatically fill the space, which basically accomplishes the same thing as the `w-100pc` you have, just wanted to point it out for reference.